src/index.css

```
@@ -2,11 +2,32 @@
 @tailwind components;
 @tailwind utilities;
 
-/* Hide scrollbar for navigation bar on mobile */
-.scrollbar-hide::-webkit-scrollbar {
-  display: none;
+/* Import our modern UI styles */
+@import './styles/globals.css';
```

Biome (2.1.2)

[error] This @import is in the wrong position.

Any @import rules must precede all other valid at-rules and style rules in a stylesheet (ignoring @charset and @layer), or else the @import rule is invalid.
Consider moving import position.

(lint/correctness/noInvalidPositionAtImportRule)